docs/contributing/events/security_alerts/bpf_attach.md

```diff
@@ -0,0 +1,62 @@
+# bpf_attach
+
+## Intro
+bpf_attach - a BPF program is attached to a probe (kprobe/uprobe/tracepoint)
+
+## Description
+An event marking that a BPF program was attached to a probe in the system.
+It occurs whenever a BPF program is attached to a perf event of the types: kprobe, uprobe or tracepoint.
+The purpose of the event is to give the user information about the BPF program, 
+as well as information about the probe itself.
+
+### Note
+The event doesn't occur for FIFO files or other files with no persistent offsets,
+to reduce spam events.
```

Comment:
It is not clear what does this mean, and how it is related to this event

Reply:
sorry, it is not related (left over from copy-paste).
should i keep the `### Note` section empty? or remove it?